### ISSUE TITLE
Tabs and toolbar are too small hard to click (should have bigger click area or should be customizable)

### ISSUE THREAD

**Issue by lednhatkhanh** [brave/browser-laptop#5683]
It's too small, sometime I need to click 2-3 times to active it

**Comment by Liunkae** [comment]
Perhaps when we get UI customization, there can be a setting to adjust the height of the tabsbar.

**Comment by valtt** [comment]
I second this. I am running Brave on Win10, screen resolution 1920x1080, "change the size of text, apps and other items setting set to 125%" and Brave tab bar and address bar itself and the fonts there are a bit too small. No such issue with Firefox or Chrome.

**Comment by 2OldMan** [comment]
The tabs and headers are way too small.  Is there a way to customize them?

**Comment by bradleyrichter** [comment]
Ahhhh.....this is a problem with HD resolution on Windows. 

@bsclifton We need to fix this Windows issue. Mac OS uses a different approach which is why there is no issue on mac HD laptop screen resolutions. 

Perhaps Electron has a pixel doubling solution?

**Comment by bsclifton** [comment]
@2OldMan there isn't a way to customize this... yet. Definitely something we'd like to get better at :smile:

**Comment by Liunkae** [comment]
Solutions to this problem: #5978.

**Comment by bradleyrichter** [comment]
Increasing the tab size should help until we can allow customization with normal and large styles for fonts and important UI controls.

**Comment by bsclifton** [comment]
@lednhatkhanh @valtt we recently pushed a new version of Brave (0.13.0) which should fix this issue for you 😄  

I believe you both are using Windows at 125% (for sure, it sounds like @valtt is).  Basically, Chromium would disregard zoom at 125% (instead rendering at 100%).  With our newest release, Chromium has fixed this behavior

If you have a few mins to test, please let us know if it resolves the issue.  Thanks! 😄

**Comment by valtt** [comment]
@bsclifton yes, the issue has been resolved now. Thanks!

**Comment by bsclifton** [comment]
@valtt awesome 😄  I'll close this issue in favor of https://github.com/brave/browser-laptop/issues/246.  I'll be sure to echo the sentiment about wanting to customize the size of tabs and font there
